datasets/ccaligned_multilingual/README.md

```diff
@@ -0,0 +1,169 @@
+---
+annotations_creators: []
+language_creators:
+- found
+languages: []
```

Comment:
Would it be possible to have the list of languages here ?

Reply:
All the languages? Or the language code given on the dataset?

Reply:
All the languages from the list on the webpage: http://www.statmt.org/cc-aligned/ :)

In terms of format we're using the BCP-47 codes. This means we use ISO-639-1 if available (`en`, `fr`, etc.). If some languages can't be formatted using an ISO-639-1 code, then you can use ISO-639-2, or ISO-639-3.
I'm pretty sure all languages for this dataset are valid ISO-639-1 code if you take what's before the `_`:
```
en_XX -> en
```